alembic/scripts/add_citation.py

```diff
@@ -0,0 +1,69 @@
+from os import environ
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from agr_literature_service.api.crud import reference_crud
+from agr_literature_service.api.models import ReferenceModel
+import datetime
+
+
+def create_postgres_session():
+    """Connect to database."""
+    USER = environ.get('PSQL_USERNAME', 'postgres')
+    PASSWORD = environ.get('PSQL_PASSWORD', 'postgres')
+    SERVER = environ.get('PSQL_HOST', 'localhost')
+    try:
+        PORT = environ.get('PSQL_PORT', 5432)
+    except KeyError:
+        PORT = '5432'
+
+    DB = environ.get('PSQL_DATABASE', 'literature')
+
+    # Create our SQL Alchemy engine from our environmental variables.
+    engine_var = 'postgresql://' + USER + ":" + PASSWORD + '@' + SERVER + ':' + PORT + '/' + DB
+    engine = create_engine(engine_var)
+
+    Session = sessionmaker(bind=engine)
+    session = Session()
+
+    return session
+
+
+def main():
+    db = create_postgres_session()
+    print("Starting {}".format(datetime.datetime.now()))
+    outfile = open("citation_update.txt", 'a')
+    seen = {}
+    i = 0
+    last_seen = 0
+    batch_size = 1000
+
+    descending = db.query(ReferenceModel).order_by(ReferenceModel.reference_id.desc())
+    last_ref = descending.first()
+    print("Last referecne_id is : {}".format(last_ref.reference_id))
```

Comment:
referecne_id -> reference_id 
not important

Reply:
I will fix typos etc later.